connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java

```diff
@@ -41,6 +41,33 @@ public class TableReadWriteTest extends BaseSparkIntegrationTest {
 
   private TableOperations tableOperations;
 
+  @Test
+  public void testNoDeltaCatalog() throws IOException, ApiException {
+    SparkSession.Builder builder = SparkSession.builder()
+            .appName("test")
+            .master("local[*]")
+            .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension");
+    String catalogConf = "spark.sql.catalog.spark_catalog";
+    builder = builder
+            .config(catalogConf, UCSingleCatalog.class.getName())
+            .config(catalogConf + ".uri", serverConfig.getServerUrl())
+            .config(catalogConf + ".token", serverConfig.getAuthToken())
+            .config(catalogConf + ".__TEST_NO_DELTA__", "true");
+    SparkSession session = builder.getOrCreate();
+
+    setupExternalParquetTable(PARQUET_TABLE, new ArrayList<>(0));
+    testTableReadWrite(SPARK_CATALOG + "." + SCHEMA_NAME + "." + PARQUET_TABLE, session);
+
+    // Since `DeltaCatalog` is not used, the following CREATE TABLE command doesn't fail while it
+    // should because the table location is empty and Delta Lake shouldn't allow it.
```

Comment:
if this is to prove that DeltaCatalog is not being used... then i think this is a very brittle test. This assumes that there is some check somewhere in a different project that can be removed any time in the future and then this test will break in such a way that we will not know how to really fix (because we wont know what to other failure to rely on)

Reply:
then I don't have a good idea to test it... Delta is still in the classpath and Spark will trigger the delta scan.

Maybe we don't need to verify it as it's obvious that `DeltaCatalog` won't be used in this test.

Reply:
maybe so.